src/components/counter.ts

```diff
@@ -7,15 +7,28 @@ import type {Extension} from '../main'
 
 export class Counter {
     private readonly extension: Extension
+    disableCountAfterSave: boolean = false
 
     constructor(extension: Extension) {
         this.extension = extension
     }
 
-    async count(file: string, merge: boolean = true) {
-        if (this.extension.manager.rootFile !== undefined) {
```

Comment:
This condition was most likely wrong. Nevertheless, we do not need the `rootFile` inside `count`. Hence no need to mark it as `async` anymore.